tests/market/test_market.py

```diff
@@ -13,9 +14,14 @@
 
 wallets = [MM_WALLET, MM_WALLET2, TERMINATE_WALLET]
 
-row_selector = '[data-testid="tab-all-markets"] .ag-center-cols-container .ag-row'
+table_row_selector = '[data-testid="tab-all-markets"] .ag-center-cols-container .ag-row'
```

Comment:
This variable name is misleading. The selector is not a table row but the container element of the whole table. Would it not be better named something like `all_markets_tab`?

Reply:
hymm, but it is  the table row selector in all markets tab:
![image](https://github.com/vegaprotocol/console-test/assets/119658839/73963069-5ac0-4fea-8c46-5b95d4727ec8)

Reply:
Think I only read the firs part of the selector.